suse/sle-hpc-15.x/sle-hpc-15-sp4/install_docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -ex
+
+# Install docker
+zypper in -y -l docker
+DOCKER_VERSION=$(rpm -q --qf="%{VERSION}" docker)
+
+# Ensure the Docker service is running
+systemctl --now enable docker
+
+# Test to run the hw container
+# docker run --rm hello-world
```

Comment:
This is added in run-tests.sh

Reply:
was only a comment how to test the docker install - not needed at all here.